certbot/tests/crypto_util_test.py

```diff
@@ -29,6 +30,9 @@ class InitSaveKeyTest(test_util.TempDirTestCase):
     def setUp(self):
         super(InitSaveKeyTest, self).setUp()
 
+        self.workdir = os.path.join(self.tempdir, 'workdir')
```

Comment:
What was the reason behind this change?

Reply:
Without it, Windows tests can fail because the owner of the directory containing the key is not the correct one (it can be the actual user or Administrators group depending on the machine). Using a dedicated directory created with `filesystem.mkdir` ensures that the folder containing the keys is properly constructed accordingly to the Windows security model, like it would be done in a real execution of Certbot, and so make the tests behave correctly on this platform.